packages/amplify-category-auth/API.md

```diff
@@ -110,7 +110,7 @@ export function initEnv(context: any): Promise<void>;
 export function isSMSWorkflowEnabled(context: any, resourceName: any): Promise<boolean>;
 
 // @public (undocumented)
-export function migrate(context: any): Promise<void>;
```

Comment:
why did we go from `Promise<void>` to `Promise<any>` ?

Reply:
Changed this back to Promise<void>